evap/contributor/views.py

```diff
@@ -28,7 +28,9 @@ def index(request):
     own_courses = Course.objects.filter(
         Q(evaluations__state__in=contributor_visible_states) & (
             Q(responsibles=user) |
-            Q(evaluations__contributions__contributor=user)
+            Q(evaluations__contributions__contributor=user) |
+            Q(evaluations__contributions__contributor__in=user.represented_users.filter(is_proxy_user=True)) |
```

Comment:
please get the represented proxy users before the `own_courses` query and use the variable in the two occurrences here